### DISCUSSION TITLE
Postal Package 404 not found

### DISCUSSION THREAD

**Discussion by abdallahtaleb** [postalserver/postal#1521] [Help with using Postal] [answered]
Hi guys, 
We used this location to download postal: https://postal.atech.media/packages/stable/latest.tgz

Now it is showing not found. Any idea?

**Comment by adamcooke** [comment, marked as answer]
This installation method is no longer supported. You should follow the [new instructions](https://docs.postalserver.io).

**Reply by willpower232** [reply]
is there an official upgrade page from the Old Way to the New Way yet? 😅

**Reply by adamcooke** [reply]
Not yet. A job for tonight or tomorrow I reckon 🚀

**Reply by adamcooke** [reply]
FWIW, that `latest.tgz` package was hugely outdated anyway because we hadn't released an official build of Postal for some time.
